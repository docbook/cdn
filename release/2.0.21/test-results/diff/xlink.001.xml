<html xmlns:deltaxml="http://www.deltaxml.com/ns/well-formed-delta-v1" xmlns="http://www.w3.org/1999/xhtml" xmlns:dxx="http://www.deltaxml.com/ns/xml-namespaced-attribute" xmlns:dxa="http://www.deltaxml.com/ns/non-namespaced-attribute" deltaxml:deltaV2="A!=B" deltaxml:version="2.0" deltaxml:content-type="full-context">
   <head deltaxml:deltaV2="A=B">
      <link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css"/>
      <link rel="stylesheet" type="text/css" href="../../resources/css/default.css"/>
      <link href="../../resources/css/prism.css" rel="stylesheet" type="text/css"/>
      <link rel="stylesheet" type="text/css" href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/start/jquery-ui.css"/>
      <meta charset="utf-8"/>
      <meta name="generator" content="DocBook XSL 2.0 Stylesheets Vxxx"/>
      <script type="text/javascript" src="../../resources/js/dbmodnizr.js"/>
      <title>Inline extended XLink</title>
   </head>
   <body deltaxml:deltaV2="A!=B">
      <article deltaxml:deltaV2="A!=B" id="R.1" class="article">
         <header deltaxml:deltaV2="A=B" class="article-titlepage">
            <h2>Inline extended XLink</h2>
         </header>
         <p deltaxml:deltaV2="A">What about
<span>
               <span class="nhrefs">
                  <span class="source">
                     <em class="citetitle">DocBook</em>
                  </span> [<a href="http://docbook.org/" class="arc">DocBook.org</a>, <a href="http://en.wikipedia.org/wiki/DocBook" class="arc">DocBook on Wikipedia</a>]</span>
            </span>?
</p><div deltaxml:deltaV2="B" class="content">
            <p>What about
<span>
                  <span class="nhrefs">
                     <span class="source">
                        <em class="citetitle">DocBook</em>
                     </span> [<a href="http://docbook.org/" class="arc">DocBook.org</a>, <a href="http://en.wikipedia.org/wiki/DocBook" class="arc">DocBook on Wikipedia</a>]</span>
               </span>?
</p>
         </div>
      </article>
      <script deltaxml:deltaV2="A=B" type="text/javascript" src="http://code.jquery.com/jquery-1.6.4.min.js"/>
      <script deltaxml:deltaV2="A=B" type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"/>
      <script deltaxml:deltaV2="A=B" type="text/javascript" src="../../resources/js/nhrefs.js"/>
      <script deltaxml:deltaV2="A=B" src="../../resources/js/prism.js"/>
   </body>
</html>