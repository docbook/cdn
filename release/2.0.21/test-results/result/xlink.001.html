<html xmlns="http://www.w3.org/1999/xhtml" xmlns:p="http://www.w3.org/ns/xproc" p:style=""><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Results for xlink.001.xml</title><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link rel="stylesheet" type="text/css" href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/start/jquery-ui.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/prism.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/db-prism.css" /><link href="../style/show-results.css" rel="stylesheet" type="text/css" /><meta name="deltaxml" content="true" /></head><body><h1>DocBook XSLT 2.0 Stylesheet output: xlink.001.xml</h1><table class="diffs"><tbody><tr><th>Expected (<span class="exdiff">1</span> differences)</th><th>Actual (<span class="adiff">1</span> differences)</th></tr><tr><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/start/jquery-ui.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Inline extended XLink&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="article"</span> <span class="att">id="R.1"</span>&gt;
         <span class="tag nodiff">&lt;header <span class="att">class="article-titlepage"</span>&gt;
            <span class="tag">&lt;h2&gt;Inline extended XLink&lt;/h2&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag diff del">&lt;p&gt;What about
<span class="tag">&lt;span&gt;
               <span class="tag">&lt;span <span class="att">class="nhrefs"</span>&gt;
                  <span class="tag">&lt;span <span class="att">class="source"</span>&gt;
                     <span class="tag">&lt;em <span class="att">class="citetitle"</span>&gt;DocBook&lt;/em&gt;</span>
                  &lt;/span&gt;</span> [<span class="tag">&lt;a <span class="att">class="arc"</span> <span class="att">href="http://docbook.org/"</span>&gt;DocBook.org&lt;/a&gt;</span>, <span class="tag">&lt;a <span class="att">class="arc"</span> <span class="att">href="http://en.wikipedia.org/wiki/DocBook"</span>&gt;DocBook on Wikipedia&lt;/a&gt;</span>]&lt;/span&gt;</span>
            &lt;/span&gt;</span>?
&lt;/p&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="http://code.jquery.com/jquery-1.6.4.min.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/nhrefs.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/start/jquery-ui.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Inline extended XLink&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="article"</span> <span class="att">id="R.1"</span>&gt;
         <span class="tag nodiff">&lt;header <span class="att">class="article-titlepage"</span>&gt;
            <span class="tag">&lt;h2&gt;Inline extended XLink&lt;/h2&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag diff add">&lt;div <span class="att">class="content"</span>&gt;
            <span class="tag">&lt;p&gt;What about
<span class="tag">&lt;span&gt;
                  <span class="tag">&lt;span <span class="att">class="nhrefs"</span>&gt;
                     <span class="tag">&lt;span <span class="att">class="source"</span>&gt;
                        <span class="tag">&lt;em <span class="att">class="citetitle"</span>&gt;DocBook&lt;/em&gt;</span>
                     &lt;/span&gt;</span> [<span class="tag">&lt;a <span class="att">class="arc"</span> <span class="att">href="http://docbook.org/"</span>&gt;DocBook.org&lt;/a&gt;</span>, <span class="tag">&lt;a <span class="att">class="arc"</span> <span class="att">href="http://en.wikipedia.org/wiki/DocBook"</span>&gt;DocBook on Wikipedia&lt;/a&gt;</span>]&lt;/span&gt;</span>
               &lt;/span&gt;</span>?
&lt;/p&gt;</span>
         &lt;/div&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="http://code.jquery.com/jquery-1.6.4.min.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/nhrefs.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td></tr><tr><td><em>The “current” CSS styling is applied
                  to the this entire document; the results in this
                  column may be anomalous. Also, image links are broken
                  in this column.</em></td><td> </td></tr><tr><td><article id="R.1" class="article"><header class="article-titlepage"><h2>Inline extended XLink</h2></header><p>What about
<span><span class="nhrefs"><span class="source"><em class="citetitle">DocBook</em></span> [<a href="http://docbook.org/" class="arc">DocBook.org</a>, <a href="http://en.wikipedia.org/wiki/DocBook" class="arc">DocBook on Wikipedia</a>]</span></span>?
</p></article><script type="text/javascript" src="http://code.jquery.com/jquery-1.6.4.min.js"></script><script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script><script type="text/javascript" src="../../resources/js/nhrefs.js"></script><script src="../../resources/js/prism.js"></script></td><td><article id="R.1" class="article"><header class="article-titlepage"><h2>Inline extended XLink</h2></header><div class="content"><p>What about
<span><span class="nhrefs"><span class="source"><em class="citetitle">DocBook</em></span> [<a href="http://docbook.org/" class="arc">DocBook.org</a>, <a href="http://en.wikipedia.org/wiki/DocBook" class="arc">DocBook on Wikipedia</a>]</span></span>?
</p></div></article><script type="text/javascript" src="http://code.jquery.com/jquery-1.6.4.min.js"></script><script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script><script type="text/javascript" src="../../resources/js/nhrefs.js"></script><script src="../../resources/js/prism.js"></script></td></tr></tbody></table><h2>XML source</h2><pre data-src="../src/xlink.001.xml"></pre></body><script type="text/javascript" src="http://code.jquery.com/jquery-1.6.4.min.js"></script><script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script><script type="text/javascript" src="../../resources/js/annotation.js"></script><script type="text/javascript" src="../../resources/js/prism.js"></script></html>