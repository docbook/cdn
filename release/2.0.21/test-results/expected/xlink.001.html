<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Inline extended XLink</title><meta charset="utf-8" /><meta name="generator" content="DocBook XSL 2.0 Stylesheets V2.0.17" /><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><link rel="stylesheet" type="text/css" href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/start/jquery-ui.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link href="../../resources/css/prism.css" rel="stylesheet" type="text/css" /><link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css" /></head><body><article id="R.1" class="article"><header class="article-titlepage"><h2>Inline extended XLink</h2></header><p>What about
<span><span class="nhrefs"><span class="source"><em class="citetitle">DocBook</em></span> [<a href="http://docbook.org/" class="arc">DocBook.org</a>, <a href="http://en.wikipedia.org/wiki/DocBook" class="arc">DocBook on Wikipedia</a>]</span></span>?
</p></article><script type="text/javascript" src="http://code.jquery.com/jquery-1.6.4.min.js"></script><script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script><script type="text/javascript" src="../../resources/js/nhrefs.js"></script><script src="../../resources/js/prism.js"></script></body></html>